_conferences/flutter-conf-india-2023.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Flutter Conf India"
+website: https://flutterconf.in/
+location: India
+online: false
+
+date_start: ???
+date_end:   ???
+
+cfp:
+  start: ???
+  end:   ???
+  site: https://flutterconf.in/
```

Comment:
https://x.com/flutterconfin/status/1699039378046398585
Maybe there's a date already? On my phone so didn't find it easily

Reply:
I didn't find any on their webpage or their twitter posts :/

Reply:
@ueman Dec the 9th

Reply:
I'm currently on vacation without a computer. Feel free to edit and merge yourself :)

Reply:
Enjoy OOO 🥳